File: yht0827/1월/1월 23일/예산.java
public int solution(int[] d, int budget) {
	int answer = 0;
	int total = 0;
	Arrays.sort(d);
	for (int i = 0; i < d.length; i++) {
		if (total + d[i] <= budget) {
			total += d[i];
			answer++;
		}
	}
	return answer;
}

File: zuzubibi/old/1월/1월 23일/예산.py
def solution(d, budget):
    d. sort()
    cnt = 0
    
    for i in d:
        budget -= i

        if budget < 0:
            break

        cnt += 1
        print((budget, cnt), end=" ")
    return cnt

print(solution([1,3,2,5,4], 9)) # 3
print(solution([2,2,3,3], 10)) # 4